src/schemas/workflow.py

```diff
@@ -0,0 +1,8 @@
+from typing import TypedDict
+
+from src.models.batch_registry import SalesforceBatchRegistry
+
+
+class WorkflowOutput(TypedDict):
```

Comment:
```suggestion
class ShortHistoryBatch(TypedDict):
```
LatestHistoryBatch, RecentHistoryBatch.... (nitpick)